### PR TITLE
Adjust config test for new load_config function

### PR DESCRIPTION
**Pull request by temirrr**
Removed most of the previous code. Adjusted everything to the new `load_config` function. I decided to check in this test case whether all of the keys read from `manager.toml` file are existing in resulting dict variable.

### CODE REVIEW

File: src/ai/backend/gateway/config.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
         t.Key('addr'): tx.HostPortPair,
         t.Key('name'): tx.Slug[2:64],
         t.Key('user'): t.String,
-        t.Key('password'): t.String,
+        t.Key('password'): t.String(),
```
Comment (adrysn) [inline src/ai/backend/gateway/config.py]: Is there a reason that `t.String` is changed to `t.String()`?

File: tests/gateway/test_config.py [hunk 2]
```diff
@@ -1,49 +1,14 @@
...
+    The path for loading is automatically determined by config.read_from_file method
+    from ai.backend.common package.
+    The path is "some_path_from_root/halfstack/backend.ai-dev/manager/manager.toml"
+    in case of halfstack testing during development
+    '''
+    args = load_config()
+    for key in args.keys():
+        assert key in \
+            ('_src', 'etcd', 'db', 'manager', 'heartbeat-timeout', 'docker-registry', 'logging', 'logging.pkg-ns', 'logging.console', 'logging.file', 'logging.logstash', 'debug')
+    assert 'manager.toml' in str(args['_src'])
```
Comment (adrysn) [inline tests/gateway/test_config.py]: There are some flake8 warnings. You can check with `flake8` command.

```console
manager ❯ flake8 tests/gateway/test_config.py
tests/gateway/test_config.py:3:1: E302 expected 2 blank lines, found 1
tests/gateway/test_config.py:13:106: E501 line too long (178 > 105 characters)
tests/gateway/test_config.py:14:47: W292 no newline at end of file
```

File: src/ai/backend/gateway/config.py (hunk 1 above)
Comment (temirrr) [reply src/ai/backend/gateway/config.py]: @adrysn Oh, I didn't intend to do it. It's just some clumsy clean-up of the previous commit

**Comment by achimnol** [comment]
The test suites were completely rewritten to contain only isolated/mocked unit tests and we plan to move all integration tests to the client SDK. Closing.

**Pull request closed without merging**